### PR TITLE
Fix #32 by adding some release scripts and adding release instruction…

### LINKED ISSUE #32: Document release process
We should add the release process steps to the readme.

### PR DESCRIPTION
**Pull request by Andersos**
…s to README.md

### CODE REVIEW

**Comment by Andersos** [comment]
@schibsted/identity anyone want to have a look at this?

**Comment by linnhege** [comment]
Looks good :)

**Comment by Andersos** [comment]
These two steps probably needs more explaining:
Do a release on Github with the built files.  
Upload built files to CDN bucket.  

I am unfamiliar with what CDN bucket we are talking about in https://github.com/schibsted/sdk-js/issues/32#issuecomment-172550361

File: package.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
   ],
   "scripts": {
     "test": "grunt test",
-    "build": "npm install && grunt"
+    "build": "npm install && grunt",
+    "preversion": "npm test",
+    "version": "npm run build",
+    "postversion": "git push && git push --tags"
```
Comment (joawan) [inline package.json]: Remove one git push. Git push --tags pushes all including tags

File: README.md [hunk 2]
```diff
@@ -8,6 +8,17 @@ Used to check if user is logged in or not, or owns a product or not.
...
 
+## Release
+
+Bump [Semver](http://semver.org/) version with this command  
+
+`$ npm version <patch | minor | major>`  
+
+`$ npm publish`  
+
+Do a release on Github with the built files.
```
Comment (joawan) [inline README.md]: Manual step, but can be made with the API or by Travis.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,17 @@ Used to check if user is logged in or not, or owns a product or not.
...
+## Release
+
+Bump [Semver](http://semver.org/) version with this command  
+
+`$ npm version <patch | minor | major>`  
+
+`$ npm publish`  
+
+Do a release on Github with the built files.  
+Upload built files to CDN bucket.
```
Comment (joawan) [inline README.md]: Configure it in Travis on successful build of tag. See assets repo.

File: README.md (hunk 2 above)
Comment (joawan) [inline README.md]: https://docs.travis-ci.com/user/deployment/releases/

File: package.json (hunk 1 above)
Comment (Andersos) [inline package.json]: Yeah true =)

**Pull request merged**